### PR TITLE
Allow setting the handbook title

### PR DESCRIPTION
**Pull request by mattheu**
image

### CODE REVIEW

**Comment by mattheu** [comment]
Just a touch smaller, little more whitespace.

image

File: header.php [hunk 1]
```diff
@@ -29,16 +29,26 @@
...
-			</h1>
+			<div class="site-logo">
+
+				<div class="site-logo">
+					<a class="HMLogo HMLogo-White" href="<?php echo esc_url( home_url() ); ?>/">
+						Human Made Limited
+					</a>
+				</div>
+
+				<h1 class="site-title">Employee Handbook</h1>
```
Comment (sambulance) [inline header.php]: Does this want to be `bloginfo( 'name' )` too?

Comment (mattheu) [inline header.php]: Oh yes it should be.

**Comment by sambulance** [comment]
Good to merge once site title is updated 👍

**Pull request merged**